src/platform/packages/shared/kbn-grouping/src/containers/query/index.test.ts

```diff
@@ -93,8 +93,13 @@ describe('group selector', () => {
 
     it('groupByField script should contain logic which gets all values of groupByField', () => {
       const scriptResponse = {
-        source:
-          "def groupValues = doc[params['selectedGroup']]; int count = groupValues.size(); if (count == 0 || count > 100 ) { emit(params['uniqueValue']); } else { emit(groupValues.join(params['uniqueValue']))}",
+        source: `def groupValues = [];
```

Comment:
Super minor nit - feel free to ignore - but if the template literal isn’t aligned to avoid impacting the Painless script, we could use ``` dedent`` ``` (already installed in Kibana) to clean that up.